pyproject.toml

```diff
@@ -0,0 +1,44 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = [ "setuptools>=45", "setuptools_scm[toml]>=6.2",]
+
+[project]
+classifiers = [ "Development Status :: 2 - Pre-Alpha", "Natural Language :: English", "Programming Language :: Python :: 3",]
```

Comment:
Classifiers?

Reply:
`Development Status :: 5 - Production/Stable` is probably all I'd be comfortable with for any of our libraries, as I think about it.  Do we have anything we'd consider "mature"?

Reply:
OK, switched to production/stable 👍